### PR TITLE
bugfix: supported the encryption/decryption for multi priv keys

### PR DESCRIPTION
**Pull request by tokers**
### What this PR does / why we need it:

Previously I added the support for setting multiple certs and pkeys for a single domain, but the encryption for private keys  was forgotten, so here i fixed it.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

**Comment by moonming** [comment]
@tokers please rebase master for this PR, thx

**Comment by tokers** [comment]
@moonming OK.

**Comment by tokers** [comment]
@moonming Rebased.

File: apisix/http/router/radixtree_sni.lua [hunk 1]
```diff
@@ -62,6 +62,20 @@ local function parse_pem_priv_key(sni, pkey)
 end
 
 
+local function decrypt_priv_pkey(iv, key)
+    if str_find(key, "---") then
```
Comment (membphis) [inline apisix/http/router/radixtree_sni.lua:66]: should we use `core.string.has_prefix` here?

File: apisix/http/router/radixtree_sni.lua [hunk 2]
```diff
@@ -95,14 +109,22 @@ local function create_router(ssl_items)
...
 
             -- decrypt private key
-            if aes_128_cbc_with_iv ~= nil and
-                not core.string.has_prefix(ssl.value.key, "---") then
-                local decrypted = aes_128_cbc_with_iv:decrypt(ngx_decode_base64(ssl.value.key))
-                if decrypted == nil then
-                    core.log.error("decrypt ssl key failed. key[", ssl.value.key, "] ")
-                else
+            if aes_128_cbc_with_iv ~= nil then
+                local decrypted = decrypt_priv_pkey(aes_128_cbc_with_iv,
```
Comment (membphis) [inline apisix/http/router/radixtree_sni.lua:113]: the `ssl.value.key` maybe nil. how about this code style?

```lua
if ssl.value.key then
   ...
end

if ssl.value.keys then
    ...
end
```

**Review by membphis — CHANGES_REQUESTED** [review]
do we need to add some new test cases?

File: apisix/http/router/radixtree_sni.lua (hunk 2 above)
Comment (tokers) [reply apisix/http/router/radixtree_sni.lua:113]: sure.

File: apisix/http/router/radixtree_sni.lua (hunk 1 above)
Comment (tokers) [reply apisix/http/router/radixtree_sni.lua:66]: OK, actually i just copy and paste the original codes.

**Comment by tokers** [comment]
> do we need to add some new test cases?

Will supply some test cases.

**Review by membphis — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Pull request merged**